src/main/java/racingCar/view/InputView.java

```diff
@@ -0,0 +1,25 @@
+package racingCar.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final Scanner scanner = new Scanner(System.in);
+
```

Comment:
```suggestion
public class InputView {
    private static final Scanner scanner = new Scanner(System.in);

    private InputView() {
    }
```
객체 생성을 유도하지 않는 클래스의 경우 생성자를 막으면 좋겠네요 :)